### PR TITLE
User code namespaces page

### PR DESCRIPTION
**Pull request by Sodik**

### CODE REVIEW

File: docs/modules/ROOT/images/NamespaceGridView.png
Comment (promanenko) [inline docs/modules/ROOT/images/NamespaceGridView.png:1]: could you please add some meaningful names for namespaces?

Comment (promanenko) [inline docs/modules/ROOT/images/NamespaceGridView.png:1]: I think it makes sense to add an explanation why we can't show type for some UCNs

Comment (promanenko) [reply docs/modules/ROOT/images/NamespaceGridView.png:1]: > the exact reason is actually due to platform logic.

do you know this logic?

**Review by promanenko — APPROVED** [review]

File: docs/modules/clusters/pages/namespaces.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
+= User Code Namespaces
+:description: You can use the Namespaces menu item to monitor all the user code namespaces that are connected to your Hazelcast cluster.
```
Comment (Serdaro) [inline docs/modules/clusters/pages/namespaces.adoc:2]:
```suggestion
:description: You can use the *Namespaces* menu item to monitor all the user code namespaces that are connected to your Hazelcast cluster.
```

File: docs/modules/clusters/pages/namespaces.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+:page-aliases: ROOT:namespaces.adoc
+
+You can use the *Namespaces* menu item to monitor all the user code namespaces that are connected to your Hazelcast cluster.
```
Comment (Serdaro) [inline docs/modules/clusters/pages/namespaces.adoc:5]:
```suggestion
{description}
```

File: docs/modules/clusters/pages/namespaces.adoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+.User Code Namespace statistics overview
+[cols="20%s,80%a"]
+|===
+|Field|Description
+
+|Name
+|Name of the user code namespace.
+
+|Resource Count
+|Number of resources contained in a namespace
```
Comment (Serdaro) [inline docs/modules/clusters/pages/namespaces.adoc:18]:
```suggestion
|Number of resources contained in a namespace.
```

File: docs/modules/clusters/pages/namespaces.adoc [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+|Size
+|The total size in bytes of the resources contained in a namespace
```
Comment (Serdaro) [inline docs/modules/clusters/pages/namespaces.adoc:21]:
```suggestion
|Total size in bytes of the resources contained in a namespace.
```

File: docs/modules/clusters/pages/namespaces.adoc [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+|Type
+|The type of the resources contained in a namespace. N/A - namespace does not contain any resource yet
```
Comment (Serdaro) [inline docs/modules/clusters/pages/namespaces.adoc:24]:
```suggestion
|Type of the resources contained in a namespace. N/A - namespace does not contain any resource yet.
```

File: docs/modules/clusters/pages/namespaces.adoc [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+|Update Time
+|Update time of a user code namespace configuration
```
Comment (Serdaro) [inline docs/modules/clusters/pages/namespaces.adoc:27]:
```suggestion
|Update time of a user code namespace configuration.
```

**Review by Serdaro — COMMENTED** [review]
Minor suggestions.

File: docs/modules/clusters/pages/namespaces.adoc (hunk 2 above)
Comment (Sodik) [reply docs/modules/clusters/pages/namespaces.adoc]: Text formatting does not work in description.
image

**Comment by Serdaro** [comment]
@Sodik , we can use the `pass` argument as follows for passing the formatting to the description text:

":description: pass:q[You can use the *Namespaces* menu item to monitor all the user code namespaces that are connected to your Hazelcast cluster.]"

**Review by rebekah-lawrence — APPROVED** [review]
LGTM

**Pull request merged**